src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/Order.orm.xml

```diff
@@ -79,6 +79,7 @@
         </many-to-many>
 
         <field type="string" name="tokenValue" column="token_value" nullable="true" />
+        <field type="string" name="customerIp" column="customer_ip" nullable="true" />
```

Comment:
I've added it to `Core`, as I don't thing assigning customer IP is strictly connected to concept of `Order`s, it's more like extra functionality in *Sylius*. But of course /cc @pjedrzejewski, as I may be wrong 😄